style.css

```diff
@@ -507,8 +507,10 @@ footer {
 
 footer hr {
   margin: 0;
-  padding: 1rem;
```

Comment:
- [ ] [OPTIONAL] I highly suggest you remove the toolbar with the battery, network ... etc indicators. It is just added to the Figma to emulate a mobile device and is not part of the design. 😎️

<img src="https://user-images.githubusercontent.com/17243325/214013007-950ec84b-125e-496a-a89f-6e0db73a5ad8.png" width="400">

Reply:
Thank you so much @amejid for clarifying the topmost part of the toolbar is not part of the design. I checked other templates as well, but couldn't see that portion except in template 2. Therefore, I thought it was part of the design. I removed the portion from my design.